src/backend/access/appendonly/appendonlywriter.c

```diff
@@ -1641,6 +1683,40 @@ UpdateMasterAosegTotalsFromSegments(Relation parentrel,
 		}
 	}
 
+	if (appendonly_compaction_segno != NIL)
+	{
+		AORelHashEntryData *aoentry;
+
+		acquire_lightweight_lock();
+
+		aoentry = AORelGetOrCreateHashEntry(RelationGetRelid(parentrel));
+		Assert(aoentry);
+
+		foreach(l, appendonly_compaction_segno)
+		{
+			int			qe_segno = lfirst_int(l);
```

Comment:
Why is the variable name qe_segno? What does "qe" mean?

Reply:
renamed